packages/gatsby-theme-newrelic/README.md

```diff
@@ -231,9 +233,9 @@ module.exports = {
 };
 ```
 
-#### `gdprTracking`
+#### `gdprTrackingId`
 
-Configuration for using [`gatsby-plugin-gdpr-tracking`](https://www.gatsbyjs.com/plugins/gatsby-plugin-gdpr-tracking/) plugin with the Gatsby site. For more details on the available configuration options, visit [the documentation.](https://www.gatsbyjs.com/plugins/gatsby-plugin-gdpr-tracking/).
+Tracking Id for Google Analytics with [`gatsby-plugin-gdpr-tracking`](https://www.gatsbyjs.com/plugins/gatsby-plugin-gdpr-tracking/) plugin with the Gatsby site. For more details on Google Analytics Tracking Ids, visit [the documentation.](https://support.google.com/analytics/answer/1008080?visit_id=637396929080724679-4016043558&rd=1).
```

Comment:
```suggestion
Tracking ID for use with Google Analytics. For more details on Google Analytics Tracking Ids, visit [the documentation.](https://support.google.com/analytics/answer/1008080?visit_id=637396929080724679-4016043558&rd=1).
```

Since we are hiding the implementation details, not sure we need to mention the plugin we are using so I suggest simplifying this language just a bit.